src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -327,11 +455,32 @@ interface IRewardsCoordinator {
     function setActivationDelay(uint32 _activationDelay) external;
 
     /**
-     * @notice Sets the global commission for all operators across all avss
-     * @param _globalCommissionBips The commission for all operators across all avss
-     * @dev Only callable by the contract owner
+     * @notice Sets the default split for all operators across all avss.
+     * @param split The default split for all operators across all avss in bips.
+     * @dev Only callable by the contract owner.
+     */
+    function setDefaultOperatorSplit(uint16 split) external;
+
+    /**
+     * @notice Sets the split for a specific operator for a specific avs
+     * @param operator The operator who is setting the split
+     * @param avs The avs for which the split is being set by the operator
+     * @param split The split for the operator for the specific avs in bips.
+     * @dev Only callable by the operator
+     * @dev Split has to be between 0 and 10000 bips (inclusive)
+     * @dev The split will be activated after the activation delay
+     */
+    function setOperatorAVSSplit(address operator, address avs, uint16 split) external;
+
+    /**
+     * @notice Sets the split for a specific operator for Programmatic Incentives.
+     * @param operator The operator on behalf of which the split is being set.
+     * @param split The split for the operator for Programmatic Incentives in bips.
+     * @dev Only callable by the operator
+     * @dev Split has to be between 1000 and 10000 bips (inclusive)
```

Comment:
Stale comment - no floor

Reply:
Done in [1d6abb0](https://github.com/Layr-Labs/eigenlayer-contracts/pull/837/commits/1d6abb05b9beb3637f32674f8b00a6763c5bbed9)